libcontainer/nsenter/nsenter_test.go

```diff
@@ -21,6 +21,11 @@ type pid struct {
 	Pid int `json:"Pid"`
 }
 
+type logentry struct {
+	Msg   string
+	Level string
+}
```

Comment:
You should probably include a `json:...` annotation here.

Reply:
Done